docs/www/acls.md

```diff
@@ -56,61 +62,160 @@ redpanda:
 
 On `rpk`'s side, you can use the `--tls-key`, `--tls-cert` and `--tls-truststore` flags to have it authenticate and establish a TLS connection:
 
-```cmd
-$ rpk topic create test-topic \
-  --tls-key <path to PEM-formatted key file> \
-  --tls-cert <path to PEM-formatted cert file> \
-  --tls-truststore <path to PEM-formatted truststore file>
+```bash
+ rpk topic create test-topic \
+--tls-key <path to PEM-formatted key file> \
+--tls-cert <path to PEM-formatted cert file> \
+--tls-truststore <path to PEM-formatted truststore file>
+```
 
+```bash
 Created topic 'test-topic'.
-You may check its config with
+```
 
+You may check its config with:
+
+```bash
 rpk topic describe 'test-topic'
 ```
 
-> `rpk` defaults to connecting to `localhost:9092`. If you're connecting to a remote broker, you will need to pass `--brokers <node IP>:<kafka API port>`
+**_Note_** - `rpk` defaults to connecting to `localhost:9092`. If you're connecting to a remote broker, you will need to pass `--brokers <node IP>:<kafka API port>`
 
-> For an in-depth guide about enabling TLS and mutual TLS in redpanda check out [our guide on the subject](https://vectorized.io/blog/tls-config/).
+For an in-depth guide about enabling TLS and mutual TLS in redpanda check out [our guide on the subject](https://vectorized.io/blog/tls-config/).
 
 ### SASL/ SCRAM
 
 Redpanda also supports client authentication via SASL/SCRAM (that is, the Simple Authentication and Security Layer protocol, using the Salted Challenge Response Authentication Mechanism), which is based on usernames & passwords.
 
-To enable it, set `redpanda.enable_sasl` to `true` in the configuration file, as well as list at least one "superuser" which after created will have permissions for all operations on the clusters.
+To enable it, set `redpanda.enable_sasl` to `true` in the configuration file, as well as list at least one "superuser" which after being created will have permissions for all operations on the clusters.
+
+You config must look something like this:
 
 ```yaml
 redpanda
   
   enable_sasl: true
   superusers:
-  - admin
+    - username: admin
   # The rest of the config...
 ```
 
+<tabs>
+  <tab id="Local Redpanda">
+  You can find the configuration file here
+
+  ```bash
+  /etc/redpanda/redpanda.yaml
+  ```
+
+After you change it's value, restart Redpanda service for changes to be in effect. 
```

Comment:
Correct grammar. "it's value" sounds like you are changing the value of one parameter where here you might be changing values of multiple parameters? Maybe use, "After you change the config...".

```suggestion
After you change it's value, restart Redpanda service for changes to take effect. 
```